app/Utils/TypeUtils.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+
+namespace App\Utils;
+
+
+use Illuminate\Support\Str;
+
+class TypeUtils
+{
+
+    public static function getTypeId($name, $types)
+    {
+        $key = Str::ucfirst(Str::lower($name));
```

Comment:
Yeah, I was about to make the command change, but I didn't want to change the data in case the clients already coded it with the case in mind. I agree though it should be all lowercase, and for display, the clients can have at it 😃